src/travel-details-screens/utils.ts

```diff
@@ -125,3 +124,16 @@ function parseDateIfString(date: any): Date {
     return date;
   }
 }
+
+export function walkOrWait(leg: Leg, nextLeg?: Leg) {
```

Comment:
Could we use another name that better describes what this does? Maybe `filterSignificantLegs`?